### ISSUE TITLE
h5py文件如何生成？

### ISSUE THREAD

**Issue by hmppt** [ganji15/HiGANplus#4]
你好，HiGAN+的作者：
如果使用其他的手写数据集，那么我该如何生成所训练所需要的h5py文件？

**Comment by songkq** [comment]
@hmppt  https://github.com/ganji15/HiGANplus/blob/main/HiGAN%2B/lib/datasets.py#L205

**Comment by hmppt** [comment]
> @hmppt https://github.com/ganji15/HiGANplus/blob/main/HiGAN%2B/lib/datasets.py#L205

非常感谢！

**Comment by hmppt** [comment]
@ganji15 
Hello, I want to know what changes I need to make to the original data before running the gen_h5file function. 

If I don't make any changes to the picture, then there will be a picture like this in my hdf5 file.
![image](https://github.com/ganji15/HiGANplus/assets/43752397/6991dd2d-47e4-45b4-a9e0-f37edaaa903c)

But I printed the picture in trnvalset_words64_OrgSz.hdf5 and it is like this
![image](https://github.com/ganji15/HiGANplus/assets/43752397/cf4cb3b8-3e09-49cd-9629-eba1843d5455)

I would like to know what is causing such a difference.

My code looks like this：
``` Python
import sys
from lib.alphabet import get_lexicon, get_true_alphabet
from networks.utils import idx_to_words
import cv2
from lib.datasets import get_dataset, get_collect_fn, Hdf5Dataset
from torch.utils.data.dataloader import DataLoader
import numpy as np
collect_fn = get_collect_fn(True, sort_style=True)
train_loader = DataLoader(
            get_dataset('iam_word_org', 'trnval',
            recogn_aug=True, wid_aug=True, process_style=True),
            batch_size=8,
            shuffle=False,
            collate_fn=collect_fn,
            num_workers=4,
            drop_last=True
        )
for i, batch in enumerate(train_loader):
    real_imgs, real_img_lens, real_wids = batch['style_imgs'], batch['style_img_lens'], batch['wids']
    real_aug_imgs, real_aug_img_lens = batch['aug_imgs'], batch['aug_img_lens']
    real_lbs, real_lb_lens = batch['lbs'], batch['lb_lens']
    max_label_len = real_lbs.size(-1)
    for j in range(real_imgs.shape[0]):
        img = real_imgs[j].squeeze().numpy()
        img = (img * 255).astype(np.uint8)
        cv2.imwrite(f"image_{i}_{j}_{real_lbs[j]}.png", img)
        sys.exit(0)
```

Thanks very much!

**Comment by ganji15** [comment]
@hmppt 

It does not matter if the images and labels match. Perhaps the code ```collect_fn = get_collect_fn(True, sort_style=True)``` changes the order of the input data.
